wagtail/admin/templates/wagtailadmin/panels/multi_field_panel_child.html

```diff
@@ -1,5 +1,5 @@
 {% load wagtailadmin_tags %}
-<div class="w-panel__wrapper {{ child.classes|join:' ' }}">
+<div class="w-panel__wrapper {{ child.classes|join:' ' }} {% include "wagtailadmin/shared/attrs.html" with attrs=self.attrs %}">
     {% if child.heading %}
```

Comment:
@salty-ivy there is a bug/typo in the quote position. it should be: 

```<div class="w-panel__wrapper {{ child.classes|join:' ' }}" {% include "wagtailadmin/shared/attrs.html" with attrs=self.attrs %}>```

Reply:
typo! will fix it.